base_layer/core/src/chain_storage/lmdb_db/lmdb_db.rs

```diff
@@ -2888,9 +2903,9 @@ fn run_migrations(db: &LMDBDatabase) -> Result<(), ChainStorageError> {
     );
     drop(txn);
 
-    for i in last_migrated_version..=MIGRATION_VERSION {
+    for migrate_from_version in last_migrated_version..=MIGRATION_VERSION {
```

Comment:
nit: for small loops i always reads easier

Reply:
For this function it clarifies the code